pkg/render/logstorage.go

```diff
@@ -440,7 +441,7 @@ func (es elasticsearchComponent) elasticsearchExternalService() *corev1.Service
 	return &corev1.Service{
 		TypeMeta: metav1.TypeMeta{Kind: "Service", APIVersion: "v1"},
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      ElasticsearchServiceName,
+			Name:      ESGatewayServiceName,
```

Comment:
I don't really like that there's a reference in the Elasticsearch component specifically for the gateway here, but I think it might be more of a hassle to change this now. I guess when we refactor the logstorage render components we can tackle this, maybe separating the external service (and possibly any other managed cluster specific components) into their own component.